expcore/Mini_games.lua

```diff
@@ -259,24 +250,22 @@ Gui.element(function(_,parent,name)
         mini_game.gui(parent)
     end
 end)
-  
+
 mini_game_list =
-Gui.element(function(event_trigger,parent,...)
+Gui.element(function(event_trigger,parent)
     local container = Gui.container(parent,event_trigger,200)
-    local header = Gui.header(
-        container,
-        "Start a game",
-        "You can start the game here.",
-        true
-    )
+
+    Gui.header(container,"Start a game","You can start the game here.",true)
+
     local scroll_table = Gui.scroll_table(container,250,3,"thing")
-    local scroll_table_style = scroll_table.style 
+    local scroll_table_style = scroll_table.style
     scroll_table_style.top_cell_padding = 3
     scroll_table_style.bottom_cell_padding = 3
-    
-    for i,e in pairs(Mini_games.mini_games) do
-        add_mini_game(scroll_table,i) 
+
+    for i,_ in pairs(Mini_games.mini_games) do
+        add_mini_game(scroll_table,i)
     end
```

Comment:
This can just be `for i in pairs` but really it should be `for name in pairs` since these are the names of mini games.